ajax/libs/sticky-kit/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "sticky-kit",
+  "version": "1.1.3",
+  "description": "A jQuery plugin for creating smart sticky elements",
+  "filename": "sticky-kit.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/leafo/sticky-kit.git"
+  },
+  "author": {
+    "name": "leafo",
+    "email": "leafot@gmail.com"
+  },
+  "license": "MIT",
```

Comment:
It's a little bit hard for them because they can't change the files contain every version from npm as I knew from here https://github.com/react-native-training/react-native-elements/issues/375.